### PR TITLE
Ref #391 -- Send email on cancelled order

### PR DESCRIPTION
**Pull request by danlls**
Send a notification email to user when
order is cancelled.

### CODE REVIEW

File: src/pretix/base/services/orders.py [hunk 1]
```diff
@@ -190,6 +190,18 @@ def _cancel_order(order, user=None):
         if position.voucher:
             Voucher.objects.filter(pk=position.voucher.pk).update(redeemed=F('redeemed') - 1)
 
+    if send_mail:
+        with language(order.locale):
+            mail(
+                order.email, _('Order cancelled for you order: %(code)s') % {'code': order.code},
```
Comment (raphaelm) [inline src/pretix/base/services/orders.py]: I think that's kinda duplicate, ``Order cancelled: %(code)s`` should be enough

File: src/pretix/base/settings.py [hunk 2]
```diff
@@ -319,6 +319,15 @@
 Best regards,
 Your {event} team"""))
     },
+    'mail_text_order_cancelled': {
+        'type': LazyI18nString,
+        'default': LazyI18nString.from_gettext(ugettext_noop("""Hello,
+
+your order: {code} for {event} has been cancelled.
```
Comment (raphaelm) [inline src/pretix/base/settings.py]: I would still include
```
You can view the details of your order at
{url}
```

**Comment by danlls** [comment]
@raphaelm amended as requested! Let me know if there is anything else that I can improve on.

File: src/pretix/control/templates/pretixcontrol/event/mail.html [hunk 3]
```diff
@@ -35,6 +35,10 @@
 
                 {% blocktrans asvar title_waiting_list_notification %}Waiting list notification{% endblocktrans %}
                 {% include "pretixcontrol/event/mail_settings_fragment.html" with pid="waiting_list" title=title_waiting_list_notification items="mail_text_waiting_list" %}
+
+                {% blocktrans asvar title_order_cancelled %}Order Cancelled{% endblocktrans %}
```
Comment (raphaelm) [inline src/pretix/control/templates/pretixcontrol/event/mail.html]: lowercase 'c', please

File: src/pretix/base/settings.py (hunk 2 above)
Comment (raphaelm) [inline src/pretix/base/settings.py]: I'd drop the ':' after 'order'

**Comment by danlls** [comment]
Done :)

**Comment by raphaelm** [comment]
Thank you very much and congratulations to your first contribution! :tada:

**Pull request merged**